.github/workflows/quality-check.yaml

```diff
@@ -5,6 +5,14 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v2
+      - uses: actions/checkout@v2
+        with:
```

Comment:
do these need to be doubled-up?

Reply:
One checks out the actual deepsparse repo and the other checks out the sparsezoo repo to a subdirectory